Wikipedia/Localizations/ur.lproj/Localizable.strings

```diff
@@ -101,7 +101,7 @@
 "search-recent-clear-confirmation-sub-heading" = "اس عمل کو واپس نہیں کیا سکتا!";
 "search-recent-clear-cancel" = "منسوخ";
 "search-recent-clear-delete-all" = "سبھی کو حذف کریں";
-"search-result-redirected-from" = "%s سے رجوع مکرر ہوا ہے";
```

Comment:
@joewalsh 
I also logged in to https://translatewiki.net and changed it there. 

If a change is tiny like this one usually there's no issue, but sometimes we need to chat with the volunteer translators before undoing or modifying their translations. You can see an example of such a discussion I had on this translator's talk page: (i'm `mhurd` on the talk page)
https://translatewiki.net/wiki/User_talk:Purodha